bgpd/bgp_bmp.c

```diff
@@ -1271,7 +1298,7 @@ static struct bmp *bmp_open(struct bmp_targets *bt, int bmp_sock)
 
 	set_nonblocking(bmp_sock);
 	set_cloexec(bmp_sock);
-	shutdown(bmp_sock, SHUT_RD);
+	//shutdown(bmp_sock, SHUT_RD);
```

Comment:
is this intentional?

Reply:
My colleague said it was needed. But I commented it rather than deleting it, to give others here with more knowledge the chance to decide if it is.. Why would you want to shutdown() during bmp_open()?

Reply:
my question was more about: why commenting and not removing the line?

Reply:
https://pubs.opengroup.org/onlinepubs/009695399/functions/shutdown.html
'SHUT_RD' disables further receive operations on the socket, and this fix is based on being able to read.. So yes 😀

Reply:
Will remove when I redo this 👍